xbmc/utils/XTimeUtils.h

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#if !defined(TARGET_WINDOWS)
+#include "PlatformDefs.h"
+#endif
+
+namespace KODI
+{
+namespace TIME
+{
+struct SystemTime
+{
+  unsigned short year;
+  unsigned short month;
+  unsigned short dayOfWeek;
+  unsigned short day;
+  unsigned short hour;
+  unsigned short minute;
+  unsigned short second;
+  unsigned short milliseconds;
+};
+
+struct TimeZoneInformation
+{
+  long bias;
+  wchar_t standardName[32];
```

Comment:
Since we don't need to care about the windows definition of the struct these should probably be std::string or at least char